SYCL/SpecConstants/2020/handler-api.cpp

```diff
@@ -23,8 +23,10 @@
 
 constexpr sycl::specialization_id<int> int_id;
 constexpr sycl::specialization_id<int> int_id2(2);
-constexpr sycl::specialization_id<double> double_id(3.14);
 constexpr sycl::specialization_id<custom_type> custom_type_id;
+#ifdef ENABLE_FP64
+constexpr sycl::specialization_id<double> double_id(3.14);
+#endif
```

Comment:
I don't think we really need to move it to after `custom_type_id` (imagine we'll remove `#ifdef` without moving it back), but I can live with it.

Same below.

Reply:
Moved `fp64` code block before `custom_type_id`